### DISCUSSION TITLE
Wifi scheduler commands

### DISCUSSION THREAD

**Discussion by szyper-sakrona** [emsesp/EMS-ESP32#2170] [Q&A]
Hello,

I noticed that if my home WIFI connection have some problems (power up/down etc) I have to manally reset device. Could you point me to create a scheduler condition like ex. "wifi status broken" "system/restart" - what do you suggest?
Thanks!

**Comment by proddy** [comment]
moving to a dicussion

**Comment by proddy** [comment]
There's an example similar to this on the demo site: https://demo.emsesp.org/scheduler

**Reply by szyper-sakrona** [reply]
superb! thanks

**Reply by MichaelDvP** [reply]
Hmm, there are some issues with the scheduler commands. 
- `system/network/network` shows `WiFi` if connected, but is not available when disconnected, we should We should set to disconnected for triggering an condition `system/network/network == disconnected`
- We can't trigger `!= WiFi` because it's camelCase but conditions convert to lower case.

I've fixed in https://github.com/MichaelDvP/EMS-ESP32/releases/tag/test
You can try and check if it works with something like:
![grafik](https://github.com/user-attachments/assets/1caa7eeb-0382-47b2-9fe9-7c24f481027f)
